extensions/mssql/src/objectExplorerNodeProvider/hdfsCommands.ts

```diff
@@ -66,8 +66,8 @@ export class UploadFilesCommand extends ProgressCommand {
 		super('mssqlCluster.uploadFiles', prompter, appContext);
 	}
 
-	protected async preExecute(context: ICommandViewContext | ICommandObjectExplorerContext, args: object = {}): Promise<any> {
-		return this.execute(context, args);
+	protected async preExecute(sqlContext: ICommandViewContext | ICommandObjectExplorerContext, args: object = {}): Promise<any> {
```

Comment:
Again please undo name changes

Reply:
rolled back.

Reply:
Still shows as `sqlContext` can you fix again?